infra/modules/gcp-host/main.tf

```diff
@@ -90,6 +90,7 @@ resource "google_secret_manager_secret_iam_member" "grant_sa_updater_on_lockable
   member    = "serviceAccount:${google_service_account.api_connectors[each.value.instance_id].email}"
   role      = module.psoxy.psoxy_instance_secret_locker_role_id
 
+  depends_on = [module.secrets]
```

Comment:
That was the missing dependency that when creating a secret for the first time, it was complaining